Week1/HTML.md

```diff
@@ -0,0 +1,260 @@
+
+  - Change the elements in the browser from the inspector, delete the current `h1` tag and add a `div` tag.
+
+    Find out more about [`div`](https://www.w3schools.com/tags/tag_div.asp)
+
+    _Notice_ that these changes will not be saved in your file and only affect the browser, if you wish to save them then you will have to actually do it from the file itself.
+
+- Add some more elements to in your `div`, after each step check your browser to see the changes:
+
+  - Add the `h1` tag back to your code with the content `The Main Heading`.
+  - Add an `h2` tag that has the content `Sunheading`.  
+  
+    Why are headings [important?](https://www.w3schools.com/html/html_headings.asp)
+  - Add a `p` tag with the content `First paragraph`.
+  - Add an `img` tag with an image from your laptop of your choosing.  
+  
+    More about [`img`](https://www.w3schools.com/tags/tag_img.asp).
+  - Add a second `div` (nested div).
+  - In the second `div` add another `p` tag with the content `Second paragraph`.  
+    
+    Comments are used to leave messages for other devs without affecting the code itself. To add a comment in an html           file use the following syntax: `<!--` your comment goes in here `-->`
+  - Now add a comment to your file just below the main heading.
+  
+  - The result should look something like this (_only use this in case you are stuck_):
+
+    <details>
+     <summary>Code result</summary>
+       &lt;div&gt;
+       <br>
+        &lt;h1&gt;The Main Heading&lt;/h1&gt;
+        <br>
+        &lt;!-- My smart comment! --&gt;
+        <br>
+        &lt;h2&gt;Subheading&lt;/h2&gt;
+        <br>
+        &lt;p&gt;Paragraph&lt;/p&gt;
+        <br>
+        &lt;img src="/" alt="Image"&gt;
+        <br>
+        &lt;div&gt;
+        <br>
+          &lt;p&gt;Second Paragraph&lt;/p&gt;
+          <br>
+        &lt;/div&gt;
+        <br>
+      &lt;/div&gt;
+    </details>
+
+      <details>
+      <summary>Result in browser</summary>
+        <div>
+          <h1>The Main Heading</h1>
+          <h2>Subheading</h2>
+          <p>Paragraph</p>
+          <img src="/" alt="Image">
+          <div>
+            <p>Second Paragraph</p>
+          </div>
+        </div>
+      </details>
+      
+### Task 2: Add more elements to your HTML page  
+
+- Read about ways of creating [ordered](https://www.w3schools.com/tags/tag_ol.asp) and [unordered](https://www.w3schools.com/tags/tag_ul.asp) lists in html. In your file add two new `div`s and create one list in each of them. The lists should look like this:  
+
+My to-do list:
+1. (_insert your tasks here_) 
+2.  
+3.  
+
+  Three people sitting next to me:  
+  - (_insert their names_)
+  -  
+  -  
+
+### Task 3 - Build a webpage to match the mockup  
+
+Create a new HTML file (let's call it exercise.html) and try to create a webpage that will look exactly like this mockup (you can click on the image to enlarge it). Mind the instructions.
```

Comment:
`(you can click on the image to enlarge it)` you can't :man_shrugging:

Reply:
Well, _I_ can.

Reply:
```suggestion
Create a new HTML file (let's call it exercise.html) and try to create a webpage that will look exactly like this mockup (you can click on the image to open it in a new tab, if you'd like to see a larger version of it). Mind the instructions.
```